www/index.html

```diff
@@ -103,66 +121,70 @@
         function geolocationError(error) {
             console.warn('ERROR(' + error.code + '): ' + error.message);
 
-            var toast = document.querySelector('paper-toast');
-            toast.text = 'something went wrong with your geolocation, please try to activate it or authorize the app and retry :)';
+            var toast      = document.querySelector('paper-toast');
+            toast.text     = 'something went wrong with your geolocation, please try to activate it or authorize the app and retry :)';
             toast.duration = 5000;
```

Comment:
I think that we maybe reuse it so I let all the markup blank and I set it dynamically in js. Like this we can let a html quite clean and play with the different elements when we want to, setting them into js . What do you think about it ?